tests/conftest.py

```diff
@@ -201,3 +201,54 @@ def _f(_addr, _salt, _initcode):
         return keccak(prefix + addr + salt + keccak(initcode))[12:]
 
     return _f
+
+
+@pytest.fixture
+def side_effects_contract(get_contract):
+    def generate(inputs):
+        """
+        Takes in a list of tuple of (
+            function_name: Function to be called by the test contract
+            return_type: Return type of the function
+            return_value: Return value of the function
+        ) and generates a Vyper contract for testing side effects.
+        """
+        code = ""
+
+        for i in inputs:
+            assert len(i) == 3
```

Comment:
i think the assert is actually not needed because otherwise the unpacking will fail on the next line